packages/ros2-bridge/src/authenticator.d.ts

```diff
@@ -0,0 +1,6 @@
+import { WebSocketMiddleware } from './websocket-connect';
```

Comment:
this should be removed, it's probably due to stale commit, can you rebase main? `git rebase main next ui/header-bar`, "next" is the commit that you initially based this PR on.

Reply:
I wasnt able to run this command; it didnt give any changes or output.. but I managed to cherry-pick the changes, so I think it should be resolved..